### DISCUSSION TITLE
XKit "branding" in added elements like modals

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#521] [Suggestion Box]
I'm not really sure how this would work (I thought of incorporating the purple theme color all over, but that sounds very intrusive), but due to XKit Rewritten's almost-too-successful imitation of the native Tumblr UI, I've been trying to think of ways to add XKit-specific or script-specific branding to things like modals and meatball items. Low-opacity monochrome script icons, for example, or some subtext bearing the script name.

(Certainly this thought is partially due to the Anti-Capitalism modal, which in the absence of something consistent like this should probably say "Anti-Capitalism" somewhere. But even when there's no ambiguity, the clarity seems like it would be nice in my opinion, if there were a nice visual style for it.)

**Comment by JRHB314** [comment]
I like this idea. As someone who uses Stylish, Tampermonkey, and some features from the old "New Xkit", it can be hard to remember sometimes what feature is coming from where.

I agree color-coding would be too intrusive. In some places having a little text marking "XKit" would work, like there already is for the Tag Replacer. Maybe just hover text in other cases. I imagine that would be fairly easy to add to the Quote Reply button, or on Timestamps, for instance.

I don't understand your second point about Anti-Capitalism though?
